build_tools/github/build_minimal_windows_image.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+set -x
+
+PYTHON_VERSION=$1
+BITNESS=$2
+
+if [[ "$PYTHON_VERSION" == "36" || "$BITNESS" == "32" ]]; then
+    # Python 3.6 and 32-bit architectures are not
+    # yet supported by the official Docker images
+    exit 0
+fi
+
+if [[ "$PYTHON_VERSION" == "37" ]]; then
+    WHEEL_NAME=cp"$PYTHON_VERSION"m-win_amd64.whl
+else
+    # Different name for Python 3.8 and newer
+    WHEEL_NAME=cp"$PYTHON_VERSION"-win_amd64.whl
+fi
+
+WHEEL_NAME=scikit_learn-$SCIKIT_LEARN_VERSION-cp$PYTHON_VERSION-$WHEEL_NAME
+
+# Find the repaired wheel because there is no a way
+# to access to the path in a straightforward manner
+WHEEL_PATH="C:/Users/RUNNER~1/AppData/Local/Temp"
+WHEEL_PATH=$(find $WHEEL_PATH -type d -name "repaired_wheel")
+WHEEL_PATH=$(realpath $WHEEL_PATH)
+WHEEL_PATH="$WHEEL_PATH/$WHEEL_NAME"
+
+cp $WHEEL_PATH $WHEEL_NAME
+
+# Dot the Python version for identyfing the base Docker image
+PYTHON_VERSION=$(echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1:2})
```

Comment:
`${PYTHON_VERSION:1:2}`: we might get a problem when Python 3.10 is out :)

I believe the following should work:

```suggestion
PYTHON_VERSION=$(echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1})
```

Reply:
Both, `PYTHON_VERSION=310; echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1:2}` and `PYTHON_VERSION=310; echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1}` return `3.10`.

Nevertheless, if you prefer `PYTHON_VERSION=$(echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1})` I would apply the suggested changes.

Reply:
Bash is really not intuitive...